.babelrc

```diff
@@ -1,3 +1,9 @@
 {
-  "stage": 1
-}
\ No newline at end of file
+  "plugins": [
+    ["transform-runtime", { "polyfill": true, "regenerator": true }]
+  ],
+  "presets": [
+    "es2015",
+    "stage-0"
+  ]
```

Comment:
Stage 0. If I don't use it then I'll need to remove object spreading, bind operator, and a couple other things I think.